### PR TITLE
Add Events to Math Magician

### PR DESCRIPTION
**Pull request by Mucyosoda**
**Math Magician**
Added event to Math Magician project so that our calculator can do different operations 
this project is built with :
- javascript
-  react
- linter
- babelrc

### CODE REVIEW

**Review by mikenath223 — CHANGES_REQUESTED** [review]
Hello  

You've done pretty well so far!
But for a few issues, you are almost done with this. Hang in there!


## Required Changes ♻️
_Please check the comments under the review._


Happy coding, Cheers! 🚀🚀

Do feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, whether it is valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/App.js [hunk 1]
```diff
@@ -1,13 +1,16 @@
+import React from 'react';
+// eslint-disable-next-line import/extensions
```
Comment (mikenath223) [inline src/App.js]: - [ ] Kindly remove this eslint disable line it's best to fix the issue in the code.

File: src/App.js [hunk 2]
```diff
@@ -1,13 +1,16 @@
...
-        <h1>Math Magician</h1>
-      </header>
-    </div>
-  );
+// eslint-disable-next-line react/prefer-stateless-function
+class App extends React.Component {
+  render() {
+    return (
+      <div className="App">
+        <DoCalculation />
```
Comment (mikenath223) [inline src/App.js]: - [ ] Currently running the app fails with the error: `Module not found: Can't resolve './components/calculator' in '/home/mike/Documents/Projects/Math-Magician/src'`, kindly fix it.

File: src/App.test.js [hunk 3]
```diff
@@ -1,7 +1,9 @@
+/* eslint-disable */
```
Comment (mikenath223) [inline src/App.test.js]: - [ ] Kindly remove this eslint disable rule.

File: src/index.js [hunk 4]
```diff
@@ -1,13 +1,12 @@
-/* eslint-disable no-unused-vars */
-
+/* eslint-disable */
```
Comment (mikenath223) [inline src/index.js]: - [ ] Kindly remove this eslint disable rule.

File: src/reportWebVitals.js [hunk 5]
```diff
@@ -1,8 +1,7 @@
-const reportWebVitals = (onPerfEntry) => {
+/* eslint-disable */
```
Comment (mikenath223) [inline src/reportWebVitals.js]: - [ ] Kindly remove this eslint disable rule.

File: src/setupTests.js [hunk 6]
```diff
@@ -1,3 +1,4 @@
+/* eslint-disable */
```
Comment (mikenath223) [inline src/setupTests.js:1]: - [ ] Kindly remove this eslint disable rule.

File: src/App.js [hunk 7]
```diff
@@ -1,13 +1,15 @@
...
-
-function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
-        <h1>Math Magician</h1>
-      </header>
-    </div>
-  );
+// eslint-disable-next-line react/prefer-stateless-function
```
Comment (afanimartin) [inline src/App.js]: Always make it a habit to fix linter errors. Please fix that error.

**Review by afanimartin — CHANGES_REQUESTED** [review]
Hi @Mucyosoda, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

### Martin Afani
#### Slack: whiz25

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/App.js (hunk 7 above)
Comment (Mucyosoda) [reply src/App.js]: Hello afanimartin, I have used that line because in first project requirement was not to use function

**Review by DeliceLydia — APPROVED** [review]
# Congratulations🎉🎉

## Hi @Mucyosoda, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard).. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**